modules/voice.js

```diff
@@ -0,0 +1,256 @@
+          }
+        ];
+
+        await channel.permissionOverwrites.set(overwrites);
+        if (channel.name.includes("[STREAM]")) channel.setName(channel.name.replace("[STREAM]", "Room"), "Channel Unlock");
+        interaction.reply({ content: "Channel successfully unlocked!", ephemeral: true });
+
+      } else {
+        interaction.reply({ content: "You don't have permission to unlock the channel right now.", ephemeral: true });
+      }
+    } else if (channel) {
+      interaction.reply({ content: "You're not in a channel that's unlockable.", ephemeral: true });
+    } else {
+      interaction.reply({ content: "You're not in a voice channel!", ephemeral: true });
+    }
+  } catch (e) { u.errorHandler(e, "Unlock Voice Channel"); }
+}
+
+async function slashVoiceStreamlock(interaction) {
+  try {
+    const member = interaction.member;
+    const channel = member.voice.channel;
+    if (isCommunityVoice(channel)) {
+      if (channel.permissionsFor(member).has("STREAM")) {
+        // stream lock the channel
+        const users = Array.from(channel.members.keys()).concat(getIDsFromMentionString(interaction.options.getString("users")));
+
+        const overwrites = [
+          { // bot
+            id: interaction.client.user.id,
+            allow: "CONNECT"
+          }, { // @everyone
+            id: interaction.guild.id,
+            deny: ["SPEAK", "STREAM"]
+          }, { // Muted
+            id: sf.roles.muted,
+            deny: ["CONNECT", "SPEAK", "VIEW_CHANNEL"]
+          }
+        ].concat(users.map(id => ({
+          id,
+          allow: ["CONNECT", "STREAM", "SPEAK"]
+        })));
+
+        await channel.permissionOverwrites.set(overwrites);
+        if (channel.name.includes("Room")) await channel.setName(channel.name.replace("Room", "[STREAM]"), "Stream Lock");
+        interaction.reply({ content: "Channel successfully stream locked!", ephemeral: true });
+      } else {
+        interaction.reply({ content: "You don't have permission to stream lock this channel.", ephemeral: true });
+      }
+
+    } else if (channel) {
+      interaction.reply({ content: "This channel cannot be stream locked!", ephemeral: true });
+    } else {
+      interaction.reply({ content: "You are not in a voice channel!", ephemeral: true });
+    }
+  } catch (e) { u.errorHandler(e, "Lock Voice Channel"); }
+}
+
+const Module = new Augur.Module()
+.addInteractionCommand({
+  name: "voice",
+  guildId: sf.ldsg,
+  commandID: sf.commands.voice,
+  process: async (interaction) => {
+    switch (interaction.options.getSubcommand(true)) {
+    case "lock":
+      await slashVoiceLock(interaction);
+      break;
+    case "unlock":
+      await slashVoiceUnlock(interaction);
+      break;
+    case "streamlock":
+      await slashVoiceStreamlock(interaction);
+      break;
+    }
+    // Refresh channels, make sure they're ok.
+    await ensureVoiceChannelsOpen(interaction.client);
```

Comment:
Let's give this its own subcommand instead of running after `lock`, `unlock`, or `streamlock`